### ISSUE TITLE
Mobile color contrast issue

### ISSUE THREAD

**Issue by bkardell** [web-platform-tests/wpt.fyi#4543]
In the mobile display of interop, the new header 'totals' lack sufficient color contrast

**Comment by foolip** [comment]
Here's a screenshot:

<img width="587" height="269" alt="Image" src="https://github.com/user-attachments/assets/5da2ff8b-9101-4acb-bddd-a0c6bc4e4d36" />

It's the same background and foreground as in the big numbers in circles at the top, but when the text is smaller the perceived contrast isn't as good.

Perhaps we can use black text or black text with 90% opacity to get better contrast? (I'm not too sure about the red, there it's possible that white contrasts better.)
